gammapy/maps/hpx/geom.py

```diff
@@ -140,10 +140,15 @@ def _create_lookup(self, region):
     def local_to_global(self, idx_local):
         """Compute a local index (partial-sky) from a global (all-sky) index.
```

Comment:
HEre the docstring suggests the opposite of what the parameters and return statement say